lib/dpl/provider/elastic_beanstalk.rb

```diff
@@ -71,7 +71,10 @@ def version_label
       end
 
       def version_description
-        context.env['ELASTIC_BEANSTALK_DESCRIPTION'] || commit_msg
+        description = context.env['ELASTIC_BEANSTALK_DESCRIPTION'] || commit_msg
+        description.gsub!(/[^\u0009\u000A\u000D\u0020-\uD7FF\uE000-\uFFFD\u10000-\u10FFFF]/, "") and
```

Comment:
As indicated in the aws sdk issue above, we are currently waiting for the upstream documentation (http://docs.aws.amazon.com/elasticbeanstalk/latest/api/API_UpdateApplicationVersion.html#API_UpdateApplicationVersion_RequestParameters) to be updated.